src/interfaces.js

```diff
@@ -83,9 +83,15 @@ export function mapTagOperations({spec, cb = nullFn, defaultTag = 'default'}) {
       // Id already exists?
       if (typeof tagObj[id] !== 'undefined') {
         // Bump counter ( for this operationId )
-        operationIdCounter[id] = (operationIdCounter[id] || 0) + 1
+        let originalCounterValue = (operationIdCounter[id] || 1)
```

Comment:
It took me a bit to decipher the logic. Maybe clearer if refactor:

```js
const originalCounterValue = (operationIdCounter[id] || 1)
tagObj[`${id}${originalCounterValue}`] = tagObj[id]
tagObj[`${id}${originalCounterValue + 1}`] = cbResult
delete tagObj[id]
operationIdCounter[id] = originalCounterValue + 1
```

Question. Say there are 3 'abc'. The 2nd 'abc', this logic will run, both will get renamed to 'abc1' and 'abc2'. The last 'abc', it will still be 'abc', as opposed to 'abc3'. Is this expected?